malaria24/ona/models.py

```diff
@@ -18,3 +25,62 @@ class ReportedCase(models.Model):
     _id = models.CharField(max_length=255)
     _uuid = models.CharField(max_length=255)
     _xform_id_string = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class EHP(models.Model):
+    """
+    An EHP is an Environmental Health Practitioner
+    """
+    name = models.CharField(max_length=255)
+    email_address = models.EmailField(null=True)
+    phone_number = models.CharField(max_length=255)
+    facility_code = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+class SMS(models.Model):
+    """
+    An SMS sent from the system, for audit trail purposes.
+    """
+    to = models.CharField(max_length=255)
+    content = models.CharField(max_length=255)
+    message_id = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+
+def alert_new_case(sender, instance, created, **kwargs):
+    from malaria24.ona.tasks import send_sms
```

Comment:
Doing the import here to prevent circular references between tasks & models while trying to import stuff.